### ISSUE TITLE
Autoinstall with automatic rebooting tricky to use

### ISSUE THREAD

**Issue by nivekuil** [coreos/coreos-installer#820]
# Feature Request #

Creating an auto installing iso is convenient but is a bit tricky to use. I tried it on a VPS and after the live iso installs, it shuts down but the VM is then rebooted, and the auto install will begin again. If the ISO is ejected during this time it will result in a corrupted host disk and the operator will have to watch the console or have some low level automation to properly time this. It would be nice if the auto install feature could tell if the host install was freshly made and refused to install in that case to ease this burden.

**Comment by bgilbert** [comment]
Good point.  The feature was originally designed for IPMI virtual CD scenarios where the host can be told to boot from CD exactly once.  What VPS are you using?

Refusing to install isn't sufficient, since we'd just boot loop in that case.  We need to skip booting from CD a second time.  It might be possible to accomplish that with bootloader tricks, maybe by checking for the "freshly-installed" disk GUID.  That's harder on BIOS, though, because we're using isolinux instead of grub there.

The fix would be in the OS, not in coreos-installer itself, so moving this to fedora-coreos-tracker.
